### PR TITLE
[Intel MKL] Fixing _MklQuantizeV2 rewrite issue

### PR DESCRIPTION
**Pull request by mahmoud-abuzaina**
This fixes _MklQuantizeV2 rewrite issue by adding 2 new attributes that were added recently to the original QuantizeV2 Op.

### CODE REVIEW

File: tensorflow/core/graph/mkl_layout_pass_test.cc [hunk 1]
```diff
@@ -1494,6 +1494,61 @@ TEST_F(MklLayoutPassTest, NodeRewrite_QuantizeV2Op_Negative_MinFirst) {
...
+      "node { name: 'C' op: 'Const' "
+      " attr { key: 'dtype' value { type: DT_FLOAT } }"
+      " attr { key: 'value' value { "
+      "    tensor { dtype: DT_INT32 tensor_shape { dim { size: 1 } } "
+      "    int_val: 0 } } } }"
+      "node { name: 'D' op: 'QuantizeV2'"
+      " attr { key: 'T'                 value { type: DT_QUINT8 } }"
+      " attr { key: 'mode'              value { s: 'SCALED' } }"
+      " attr { key: 'round_mode'        value { s: 'HALF_TO_EVEN' } }"
+      " attr { key: 'narrow_range'        value { b: true } }"
```
Comment (penpornk) [inline tensorflow/core/graph/mkl_layout_pass_test.cc]: Nit: "value" is not aligned with other lines. Could you please make it aligned? Same for the one in the next test case.

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR! I have one minor formatting comment.

File: tensorflow/core/graph/mkl_layout_pass_test.cc (hunk 1 above)
Comment (mahmoud-abuzaina) [reply tensorflow/core/graph/mkl_layout_pass_test.cc]: Fixed.

**Comment by mahmoud-abuzaina** [comment]
Thanks @penpornk for the review. I have addressed your comment.

**Review by penpornk — APPROVED** [review]
Thank you, @mahmoud-abuzaina!

**Pull request merged**